typescript/sdk/src/consts/chainMetadata.ts

```diff
@@ -110,6 +110,21 @@ export const goerli: ChainMetadata = {
   finalityBlocks: 7,
 };
 
+export const optimismgoerli: ChainMetadata = {
+  id: 420, // b'op-i' interpreted as an int
```

Comment:
This comment in incorrect right? looks like just the eip-155 chain id

Reply:
Yes, let me fix that